core/src/client/test_client.rs

```diff
@@ -531,13 +531,11 @@ impl super::EngineClient for TestBlockChainClient {
 
 impl EngineInfo for TestBlockChainClient {
     fn network_id(&self) -> NetworkId {
-        // need to move common params from machine to something else
```

Comment:
I think it's better to merge this commit with the [parent](https://github.com/CodeChain-io/foundry/pull/260/commits/500294b1f41b5f3970d4eff75868904424e73fae).

Reply:
Ok